dev/provider_packages/prepare_provider_packages.py

```diff
@@ -1694,9 +1694,8 @@ def replace_content(file_path, old_text, new_text, provider_package_id):
             os.remove(temp_file_path)
 
 
-AUTOMATICALLY_GENERATED_CONTENT = (
-    ".. THE REMINDER OF THE FILE IS AUTOMATICALLY GENERATED. IT WILL BE OVERWRITTEN AT RELEASE TIME!"
-)
+AUTOMATICALLY_GENERATED_MARKER = "AUTOMATICALLY GENERATED"
+AUTOMATICALLY_GENERATED_CONTENT = f".. THE REMAINDER OF THE FILE IS {AUTOMATICALLY_GENERATED_MARKER}. IT WILL BE OVERWRITTEN AT RELEASE TIME!"
```

Comment:
```suggestion
AUTOMATICALLY_GENERATED_CONTENT = (
    f".. THE REMAINDER OF THE FILE IS {AUTOMATICALLY_GENERATED_MARKER}. "
    f"IT WILL BE OVERWRITTEN AT RELEASE TIME!"
)
```

Flake8 says this is too long.